cves/CVE-2013-2135.yml

```diff
@@ -140,12 +161,14 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this section
     by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: |
+    This commit fixes another remote code execution vulnerability that involves unintended 
+    evaluation of a request input. This is of interest because it shows there are multiple areas
+    improperly evaluating input as the templating language used by Struts.
```

Comment:
"improperly evaluating input as the templating language used by Struts." is weirdly worded, do you mean that there are multiple places where users can provide input that Struts interprets as its templating language? Also does that not make this another case of the same vulnerability being fixed somewhere else?

Reply:
Will reword. Just trying to convey that there have been multiple instances of similar (but distinct) OGNL injection vulnerabilities.

I do not consider the other referenced template language injection issue to be the same vulnerability, as the root cause and attack vector are different. It is another instance of this vulnerability _type_.